### PR TITLE
New Font: Geist Mono

### PR DESCRIPTION
**Pull request by LeviticusNelson**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making any changes to a cask, existing or new, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask --online <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask-fonts/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [x] `brew audit --new-cask <cask>` worked successfully.
- [x] `brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

### CODE REVIEW

File: Casks/font-geist-mono.rb [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  font "Geist Mono/GeistMono-Black.otf"
+  font "Geist Mono/GeistMono-Bold.otf"
+  font "Geist Mono/GeistMono-Light.otf"
+  font "Geist Mono/GeistMono-Medium.otf"
+  font "Geist Mono/GeistMono-Regular.otf"
+  font "Geist Mono/GeistMono-SemiBold.otf"
+  font "Geist Mono/GeistMono-Thin.otf"
+  font "Geist Mono/GeistMono-UltraBlack.otf"
+  font "Geist Mono/GeistMono-UltraLight.otf"
+  font "Geist Mono/GeistMonoVF.ttf"
```
Comment (bevanjkay) [inline Casks/font-geist-mono.rb:25]: Where a variable font is available generally we should only include the variable version, unless there is a specific reason the `otf` versions should be included?

Comment (LeviticusNelson) [reply Casks/font-geist-mono.rb:25]: Hi @bevanjkay, right now the variable font is kinda broken with it's weights, so it can be commented out for now.

Comment (bevanjkay) [reply Casks/font-geist-mono.rb:25]: That works 👍🏻

**Review by miccal — APPROVED** [review]

**Comment by miccal** [comment]
Thank you @LeviticusNelson.

**Pull request merged**